.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+# .github/workflows/ci.yml
```

Comment:
Please separate this file to a commit, don't mixed with coding style formatting.

Reply:
Well if we're separating anything out it'd presumably be the formatting since this pr's primarily about adding this CI file

Reply:
Hmm, GitHub didn't comment on the commit.

I originally comment this on https://github.com/Guake/guake/pull/1877/commits/a16ccc442b37578809af0888fda77c39fa5f10ff#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f, this commit was mix of coding style formatting (by black) and adding GitHub Action yml.

For what I said to separate .yml to another commit, is to prevent above commit mixed multiple intention in it (coding style formatting and migrate to GA).

Reply:
Agree to put formatting into single commit.

Reply:
Pulled black update out for now, can just redo it on a later date.